web/src/engine/websites/VortexScans.ts

```diff
@@ -43,18 +43,12 @@ type APIPages = {
 const mangaIdScript = `
     new Promise((resolve, reject) => {
        const myregexp = /"postId"\\s*:\\s*(\\d+)/i;
-        __next_f.forEach(element => {
-            const el = element[1];
-            if (el) {
-                if((myregexp.test(el)))  {
-                   const id = parseInt(el.match(myregexp)[1]);
-                   const title = document.querySelector('meta[property="og:title"]').content.trim();
-                   const slug = window.location.pathname.split('/').pop();
-                   resolve({id: id, title: title, slug: slug});
-                }
-            }
-        });
-        reject();
+       const element = __next_f.find(el => el[1] && el[1].match(myregexp));
+       if (!element) reject("Regex didnt match :/");
```

Comment:
Code after the `if` is still executed, even after the promise was already rejected.

Reply:
I guess a throw would be ok then?

Reply:
This would work